docs/architecture/metrics.md

```diff
@@ -83,4 +83,13 @@ The [FaaS Provider](/architecture/faas-provider) is the back-end API used by oth
 | `provider_http_request_duration_seconds`     | histogram  | Seconds spent serving HTTP requests | `method`, `path`, `code`   | Community Edition  |
 | `provider_http_requests_total`               | counter    | The total number of HTTP requests   | `method`, `path`, `code`   | Community Edition  |
 
-The `http_request*` metrics record the latency and statistics of `/system/*` routes. Part of this information is also recorded in the metrics for the Gateway component. The purpose of exposing separate metrics on the provider component is to show the count of calls, to show efficiency, and to show the duration for performance testing, along with errors to flag unseen issues.
\ No newline at end of file
+The `http_request*` metrics record the latency and statistics of `/system/*` routes. Part of this information is also recorded in the metrics for the Gateway component. The purpose of exposing separate metrics on the provider component is to show the count of calls, to show efficiency, and to show the duration for performance testing, along with errors to flag unseen issues.
+
+## JetStream queue-worker
```

Comment:
I've started calling this "JetStream for OpenFaaS", can we use that heading?

Then underneath explain they are emitted from the queue-worker.